docs/user-guide/writing-your-docs.md

```diff
@@ -26,8 +26,7 @@ extensions may be used for your Markdown source files: `markdown`, `mdown`,
 `mkdn`, `mkd`, `md`. All Markdown files included in your documentation
 directory will be rendered in the built site regardless of any settings.
 
-!!! note:
-
+!!! note
```

Comment:
The blank line is not a formatting error. Instead the blank line is a matter of style. I am a strong believer in requiring a bank line between each block level item. Therefore, I insist that a blank line remain here. Perhaps the style guide should make this more clear.